### PR TITLE
DOC: Fixed PR06 docstring errors in pandas.interval_range & pandas.util.hash_array

### PR DESCRIPTION
**Pull request by samyak-jn**
DOC: Fixed PR06 docstring errors in pandas.interval_range

- [ ] xref #28724 
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/util/hashing.py [hunk 1]
```diff
@@ -62,11 +62,11 @@ def hash_pandas_object(
...
     Parameters
     ----------
-    index : boolean, default True
+    index : bool, default True
         include the index in the hash (if Series/DataFrame)
-    encoding : string, default 'utf8'
-        encoding for data & key when strings
-    hash_key : string key to encode, default to _default_hash_key
+    encoding : str, default 'utf8'
+        encoding for data & key when str
```
Comment (datapythonista) [inline pandas/core/util/hashing.py]: don't replace this one, only the types

File: pandas/core/util/hashing.py [hunk 2]
```diff
@@ -62,11 +62,11 @@ def hash_pandas_object(
...
+    hash_key : str key to encode, default to _default_hash_key
```
Comment (datapythonista) [inline pandas/core/util/hashing.py]:
```suggestion
    hash_key : str, default '_default_hash_key'
```
And move the explanation to the description

File: pandas/core/util/hashing.py [hunk 3]
```diff
@@ -143,8 +143,8 @@ def hash_tuples(vals, encoding="utf8", hash_key=None):
     Parameters
     ----------
     vals : MultiIndex, list-of-tuples, or single tuple
-    encoding : string, default 'utf8'
-    hash_key : string key to encode, default to _default_hash_key
+    encoding : str, default 'utf8'
+    hash_key : str key to encode, default to _default_hash_key
```
Comment (datapythonista) [inline pandas/core/util/hashing.py]: same as before in all the rest

**Review by datapythonista — COMMENTED** [review]
thanks for the fixes @samyak-jn, but if you contribute further, please do it to the docstrings reported by the script, the ones fixed here are private and we don't care as much

File: pandas/core/util/hashing.py (hunk 1 above)
Comment (samyak-jn) [reply pandas/core/util/hashing.py]: Okay, got it, thanks won't be repeated.

File: pandas/core/util/hashing.py (hunk 2 above)
Comment (samyak-jn) [reply pandas/core/util/hashing.py]: Not sure about "move the explanation to the description", Is not it already in the Parameters description area? Should I have to add a line to describe why it is done?
Sorry, if I'm asking something lame, newbie!

**Comment by jreback** [comment]
lgtm. @datapythonista

**Comment by datapythonista** [comment]
Thanks for the fixes @samyak-jn

**Pull request merged**